lib/puppet/module.rb

```diff
@@ -58,7 +58,7 @@ def self.is_module_namespaced_name?(name)
   attr_accessor :dependencies, :forge_name
   attr_accessor :source, :author, :version, :license, :summary, :description, :project_page
 
-  def initialize(name, path, environment)
+  def initialize(name, path, environment, include_prerelease = true)
```

Comment:
That's correct. The intention is to have everything semver compliant unless told differently from PMT.